client/src/components/categorical/categorical.js

```diff
@@ -23,11 +23,16 @@ const truncateCategories = options => {
 };
 
 @connect(state => ({
-  ranges: _.get(state.controls.world, "summary.obs", null)
+  ranges: _.get(state.controls.world, "summary.obs", null),
+  categorySelectionLimit: _.get(
+    state.config,
+    "parameters.category-selection-limit",
+    globals.configDefaults.parameters["category-selection-limit"]
+  )
```

Comment:
Recommend `categoryDisplayLimit` for clarity / ease of reading

Reply:
the design intent is to have the same string as the CLI parameter, and command line params are dashed (see the CLI param proposal).   I think it will be far more confusing to add a mapping layer between the two namespaces.

Reply:
To clarify, no complaint about doing `category-display-limit`, only replacing the word `selection` with `display`, since that better describes the intention of the constant.